src/applications/disability-benefits/all-claims/pages/privateMedicalRecordsRelease.js

```diff
@@ -0,0 +1,151 @@
+import _ from '../../../../platform/utilities/data';
+import { merge } from 'lodash';
+import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import { uiSchema as addressUI } from '../../../../platform/forms/definitions/address';
+import {
+  disabilityNameTitle,
+  recordReleaseDescription,
+  limitedConsentTitle,
+  limitedConsentTextTitle,
+  limitedConsentDescription,
+} from '../content/privateMedicalRecordsRelease';
+
+import { validateZIP } from '../validations';
+import PrivateProviderTreatmentView from '../components/PrivateProviderTreatmentView';
+import { validateDate } from 'us-forms-system/lib/js/validation';
+
+const { address } = fullSchema526EZ.definitions;
+
+export const uiSchema = {
+  'ui:description': recordReleaseDescription,
+  'ui:title': disabilityNameTitle,
```

Comment:
`getDisabilityName` requires a `name` but in this case gets `formData.name`, which I don't think exists.
On that note, though - the PMR question pages don't seem to get iterated per disability, so it seems as though this title may need to be something more generic than a specific disability's name?

Reply:
@markgreenburg I'm going to check in with UX about this one. See what they want to do about it. Seems like the trend has been to just remove the title?